### DISCUSSION TITLE
Double Tab Seeking Left/Right

### DISCUSSION THREAD

**Discussion by chethavuthy** [vidstack/player#1218] [Q&A] [answered]
hi there, i would like to know that if there are any possibilities to know if the user double tab seeking left/right, are there any state to watch that is differ from `seeking` that includes seeking both double tab on video player and slide on the progress slider ?

**Comment by mihar-22** [comment, marked as answer]
Not an officially documented or public API available for this but you can try doing it like this:

**JS**

```ts
player.addEventListener("media-seek-request", event => {
  const originEvent = event.originEvent;
  if (originEvent && 'MEDIA_GESTURE' in originEvent) {
    // ...
  }
});
```

**React**

```tsx
import { type MediaSeekRequestEvent } from '@vidstack/react';

function onSeekRequest(time: number, nativeEvent: MediaSeekRequestEvent) {
  const originEvent = event.originEvent;
  if (originEvent && 'MEDIA_GESTURE' in originEvent) {
    // ...
  }
}

<MediaPlayer onMediaSeekRequest={onSeekRequest}>
```

**Reply by chethavuthy** [reply]
excuse me, may i know if we can still be able to detect if the user is double tab left or right? i really want to differentiate them, thanks.

**Reply by mihar-22** [reply]
Check if the seek request time is less than or greater than current time. 

left = `time < currentTime`  
right = `time > currentTime`

**Reply by chethavuthy** [reply]
this helps a lot, thanks man.
